spec/models/suggested_councillor_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe SuggestedCouncillor, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
I think you could have some simple tests here for the validations. They seem quite important and worth having coverage for.